Enclosures/README.md

```diff
@@ -0,0 +1,32 @@
+# WisBlock enclosures
+
+We collected some inspirations for WisBlock compatible enclosures here. These examples are tested and should make no problems on your 3D printer.
+
+## Content
+- [Evaluation Modular](#evaluation-modular)
+- [Indoor Environment](#indoor-environment)
+- [Outdoor Solar](#outdoor-solar)
+- [Community Shared Enclosures](#community-shared-enclosures)    
+
+## What you need
+All examples have fixing points for the WisBlock Base board and the WisBlock IO modules. For your convenience, we share the 3D STEP files of the RAK5005-O WisBlock Base board and the RAK5801 WisBlock IO module (largest and highest IO module) in the [WisBlock 3D](.\WisBlock-3D) folder.
+
+## Evaluation Modular
+This concept is as modular as the WisBlock itself. It has a WisBlock Base carrier, two versions of battery enclosures and a lid.    
+The idea is to have a simple enclosure for your WisBlock to use in the laboratory or in maker-spaces that holds your WisBlock Base board and gives easy access to all sensor and IO slots.    
+The next step is to have a battery enclosure (for 18650 type battery or flat Li-Ion batteries) that can easily connected to the base carrier.    
+And last not least, a lid that gives basic protection to your WisBlock if you need to take it out for testing.    
+More details can be found in the [Evaluation-Modular](./Evaluation-Modular) folder.    
+![Evaluation Modular](/assets/evaluation-modular-overview.png)    
+
+## Indoor Environment
+This idea was inspired by the [RAKBox-B3](https://docs.rakwireless.com/Product-Categories/Accessories/RAKBox-B3). As nice as the original enclosure is, it has the problem that it is too small to fit any of the WisBlock IO modules. With this new design, we get a slightly bigger enclosure in width and height, but with a reduced height. Details and 3D files can be found in the [Indoor-Environment](./Indoor-Environment) folder.    
+![Indoor Environment](/assets/indoor-environment-overview.png)    
+## Outdoor Solar 
+For outdoor usage it is required to have at least a certain level of water-proof. This concept does not have IP65 level without additional measures, but it comes quite close.    
+Most outdoor enclosures I see, specially the ones with solar panels are not very good looking. And often the solar panel is not pointing towards the sky, which gives a bad performance. This design is an approach to make something better looking and at least have the solar panel in an angel pointing towards the sky.    
```

Comment:
Most outdoor enclosures, specially the ones with solar panels are not well designed, as the solar panel is often not pointing towards the sky. It causes less efficiency in doing it's purpose. This is an approach to make a much more efficient design.

Reply:
Done